Submitted By: Greg Schafer <gschafer at zip dot com dot au>
Date: 2003-12-06
Initial Package Version: 0.13
Origin: Greg Schafer (I based it on the other 2 glade tests).
Description: Skip xgettext-glade-3 test if expat not available.

diff -Naur gettext-0.13.orig/gettext-tools/tests/xgettext-glade-3 gettext-0.13/gettext-tools/tests/xgettext-glade-3
--- gettext-0.13.orig/gettext-tools/tests/xgettext-glade-3	2003-11-02 21:11:58.000000000 +0000
+++ gettext-0.13/gettext-tools/tests/xgettext-glade-3	2003-12-05 21:53:10.690608304 +0000
@@ -5,6 +5,17 @@
 tmpfiles=""
 trap 'rm -fr $tmpfiles' 1 2 3 15
 
+tmpfiles="$tmpfiles empty.glade"
+cat <<EOF > empty.glade
+<?xml version="1.0"?>
+<GTK-Interface/>
+EOF
+
+tmpfiles="$tmpfiles xg-gl-3.pot"
+: ${XGETTEXT=xgettext}
+${XGETTEXT} -o xg-gl-3.pot empty.glade 2>/dev/null
+test $? = 0 || { rm -fr $tmpfiles; exit 77; }
+
 tmpfiles="$tmpfiles asciitable.glade2"
 cat <<EOF > asciitable.glade2
 <?xml version="1.0" standalone="no"?> <!--*- mode: xml -*-->
